packages/ai/src/stream-to-response.ts

```
@@ -50,53 +197,217 @@ export async function streamToText(
+        }
+      }
+
+      try {
+        await durability.close()
+      } catch (closeError) {
+        recordFailure(closeError, 'closing durability stream failed')
+      }
+
+      // Iterator cancellation must reject when awaited terminalization fails.
+      // eslint-disable-next-line no-unsafe-finally
+      if (failure !== undefined) throw failure.error
```

Biome (2.5.3)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)